src/datajudge/db_access.py

```diff
@@ -648,7 +657,20 @@ def get_column(
 
     if not aggregate_operator:
         selection = sa.select([column])
-        result = engine.connect().execute(selection).scalars().all()
+
+        # snowflake-specific optimization iff pandas is installed
+        if is_snowflake(engine) and pandas_available:
```

Comment:
```suggestion
        if is_snowflake(engine):
            if not pandas_available:
                print message and exit the snowflake code returning to sqlalchemy path.
```

Concerning the message, this is an option. You delay using the flag you created until here.

Reply:
I wanted to try that one as well, unfortunately, now the error message would be printed for each and every query.

Reply:
Fair enough, then you can try to import snowflake, if it succeeds you can do the pandas check and inform in case it's not present.

Reply:
@ivergara are you satisfied with this solution?

Reply:
Sure.